tests/terraform_compliance/common/test_helper.py

```diff
@@ -268,3 +269,6 @@ def test_transform_asg_style_tags_return_directly(self):
         resource_list = 'abc'
         output = transform_asg_style_tags(resource_list)
         self.assertEqual(output, 'abc')
+
+    def test_python_version_check_success(self):
+        self.assertEqual(python_version_check(), 1)
```

Comment:
No new line at EOF

Reply:
Fixed in 61b8f56

Reply:
Oh it already had one.

Reply:
There is already a new line at the EOF :)